notebook/services/contents/tests/test_manager.py

```diff
@@ -108,7 +108,7 @@ def test_checkpoint_subdir(self):
         self.assertEqual(cp_dir, os.path.join(root, cpm.checkpoint_dir, cp_name))
         self.assertEqual(cp_subdir, os.path.join(root, subd, cpm.checkpoint_dir, cp_name))
 
-    @dec.skip_win32
+    @dec.skipif(sys.platform == 'win32' and sys.version_info[0] < 3)
```

Comment:
Prior to version 3.2 [`os.symlink`](https://docs.python.org/3.5/library/os.html#os.symlink) which is required for this test only existed for unix. This enables testing on Python 3 for Windows as all supported versions have this functionality now.